server/pages/index.jade

```diff
@@ -115,6 +115,7 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 				}
 			})();
 
+		script(src='/calypso/polyfill.js')
```

Comment:
You can calculate the hash based on size of the file or its last modification date. We can skip that, too. I don't think this is going to change in the future.